server/hwapi/data_models/enums.py

```diff
@@ -17,11 +17,103 @@
 # Written by:
 #        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
 
-
 from enum import Enum
 
 
 class CertificationStatus(str, Enum):
     CERTIFIED = "Certified"
     PARTIALLY_CERTIFIED = "Partially Certified"
     NOT_SEEN = "Not Seen"
+
+
+class BusType(str, Enum):
```

Comment:
I've decided to use enums here since it's useful to have them for filtering by device or bus type
In this case, if a new type occurs on C3, we need to update our enum as well

Reply:
btw, you can check in the schema that Bus is also enum in C3: https://certification.canonical.com/api/v2/schema#tag/public-devices/operation/public_devices_list

Reply:
Can we not filter without enums? it does sound like unnecessary coupling where a C3 change to one of these two types would require change in hardware api. Unless we need more custom logic when filtering not just comparing strings

Reply:
while filtering devices by category or bus type, I want to make sure that we use the correct values and avoid situations like using `category=GPU` instead of `VIDEO` and thinking that there are no GPU devices in the table, or using `bustype=PCI` instead of `pci`. If there are breaking-changes from the C3 side, we should know about them in advance and update our client before they get landed.

Reply:
that sounds fine to me